test/PizzeriaTest.cpp

```diff
@@ -52,16 +56,46 @@ TEST_F(PizzeriaTest, completeOrderWithStubPizza)
 TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 {   
     // Given
-    PizzaMock* mock = new PizzaMock{};
-    Pizzas pizzas = {mock};
-    EXPECT_CALL(*mock, getPrice()).WillOnce(Return(40.0));
+    NiceMock<PizzaMock> mock{};
+    Pizzas pizzas = {&mock};
+    EXPECT_CALL(mock, getPrice()).WillOnce(Return(40.0));
+    EXPECT_CALL(mock, getName()).WillOnce(Return(""));
+    EXPECT_CALL(mock, getBakingTime()).WillOnce(Return(minutes(0)));
+    EXPECT_CALL(mockTimer, sleep_for(minutes(0))).Times(1);
     
     // When
     auto orderId = pizzeria.makeOrder(pizzas);
     auto price = pizzeria.calculatePrice(orderId);
+    pizzeria.bakePizzas(orderId);
+    pizzeria.completeOrder(orderId);
 
     // Then
     ASSERT_EQ(40, price);
+}
 
-    delete mock;
+TEST_F(PizzeriaTest, completeOrderForOneStubAndTwoMockPizzas) {
+    // Given
+    PizzaStub StubPizza{"STUB"};
+    StrictMock<PizzaMock> funghiMockPizza;
+    NiceMock<PizzaMock> margheritaMockPizza;
+    Pizzas pizzas{&StubPizza, &funghiMockPizza, &margheritaMockPizza};
+
+    EXPECT_CALL(funghiMockPizza, getPrice()).WillOnce(Return(40.0));
+    EXPECT_CALL(funghiMockPizza, getName()).WillOnce(Return("Funghi"));
+    EXPECT_CALL(funghiMockPizza, getBakingTime()).WillOnce(Return(minutes(5)));
+    EXPECT_CALL(margheritaMockPizza, getPrice()).WillOnce(Return(25.0));
+    EXPECT_CALL(margheritaMockPizza, getName()).WillOnce(Return("Margherita"));
+    EXPECT_CALL(margheritaMockPizza, getBakingTime()).WillOnce(Return(minutes(9))); 
+    EXPECT_CALL(mockTimer, sleep_for(minutes(1))).Times(1);
+    EXPECT_CALL(mockTimer, sleep_for(minutes(5))).Times(1);
+    EXPECT_CALL(mockTimer, sleep_for(minutes(9))).Times(1);
+
+    // When
+    auto orderId = pizzeria.makeOrder(pizzas);
+    auto price = pizzeria.calculatePrice(orderId);
+    pizzeria.bakePizzas(orderId);
+    pizzeria.completeOrder(orderId);
+
+    // Then
+    ASSERT_EQ(75, price);
```

Comment:
You have some magic values here like 40.0, 25.0, 1, 5, 9. Use constexpr and name them properly. Then you should use them in proper EXPECT_CALL or ASSERT_EQ to show connections. Which value will work in which place.